### ISSUE TITLE
curious about title

### ISSUE THREAD

**Issue by Chris75forumname** [gkrid/dokuwiki-plugin-bez#9]
Does bez stand for "Baza eliminacji zagrożeń" ??

**Comment by solewniczak** [comment]
Yes. We translated this as "Risk Elimination Base" but "Risk Management System" can be even better name.

**Comment by Chris75forumname** [comment]
No, your name is good! I recommended your plugin in the forum. Are there some indepth instructions how to setup and use the BEZ somewhere to be found? Did you publish any research about the structure and/or the use of this plugin? Is the structure of the BEZ customizable somehow? In particular, I wonder if it could be used as a slim problem reporting system without the project stuff?
